networking/hardware_networks/using-dpdk-and-rdma.adoc

```diff
@@ -0,0 +1,37 @@
+[id="using-dpdk-and-rdma"]
+= Using Virtual Functions (VFs) with DPDK and RDMA modes
+include::modules/common-attributes.adoc[]
+:context: about-sriov
+
+toc::[]
+
+You can use Single Root I/O Virtualization (SR-IOV) network hardware with the Data Plane Development Kit (DPDK) and with remote direct memory access (RDMA).
+
+[id="examples-of-using-virtual-functions"]
+== Examples of using virtual functions in DPDK and RDMA modes
+
+[IMPORTANT]
```

Comment:
Could you reuse the technology-preview.adoc here instead?

Reply:
@ahardin-rh, probably; It's originally contributed content and it may be someone pasted it in there, rather than including it, maybe because it was originally in a module.

I'll see if I can fix it.